src/main/java/org/jabref/gui/git/GitChange.java

```diff
@@ -0,0 +1,65 @@
+package org.jabref.gui.git;
+
+ import java.util.Optional;
+
+ import javafx.beans.property.BooleanProperty;
+ import javafx.beans.property.SimpleBooleanProperty;
+ import javafx.beans.property.SimpleStringProperty;
+ import javafx.beans.property.StringProperty;
+
+ import org.jabref.gui.git.entryadd.EntryAdd;
+ import org.jabref.gui.git.entrychange.EntryChange;
+ import org.jabref.gui.git.entrydelete.EntryDelete;
+
+ import org.jabref.gui.undo.NamedCompound;
+ import org.jabref.gui.util.OptionalObjectProperty;
+ import org.jabref.model.git.BibGitContext;
+
+ public sealed abstract class GitChange permits EntryAdd, EntryChange, EntryDelete {
+     protected final BibGitContext BibGitContext;
+     protected final OptionalObjectProperty<GitChangeResolver> externalChangeResolver = OptionalObjectProperty.empty();
+     private final BooleanProperty accepted = new SimpleBooleanProperty();
+     private final StringProperty name = new SimpleStringProperty();
+
+     protected GitChange(BibGitContext BibGitContext, GitChangeResolverFactory gitChangeResolverFactory) {
+         this.BibGitContext = BibGitContext;
+         setChangeName("Unnamed Change!");
```

Comment:
No exclamation marks in English - and sentence case

```suggestion
         setChangeName("Unnamed change");
```